docs/tutorials/fundamentals/viewer.md

```diff
@@ -326,9 +326,8 @@ that is in integer steps.
 ### Viewer buttons
 
 Underneath the layers list there is a row of buttons that includes the `Console`
-button that will show or hide our console that allows you to interact with a
-python kernel. Inside the console you can access the viewer using the `viewer`
-argument.
+button. This button will show or hide our console that allows you to interact with the
+python kernel, if you launch from the command line. Inside the console you can access the viewer using the `viewer` argument. Note that if you launch napari from an interactive python environment, like ipython or jupyter, then the console will be disabled in favor of using the existing interactive console.
```

Comment:
Users could also launch it from the napari bundled app executable, not the terminal.